packages/pluggableWidgets/timeline-web/src/Timeline.editorPreview.tsx

```diff
@@ -0,0 +1,79 @@
+import { TimelinePreviewProps } from "../typings/TimelineProps";
+import TimelineComponent from "./components/TimelineComponent";
+import { createElement } from "react";
+import { BasicItemType, CustomItemType, ItemType } from "./Timeline";
+import { dynamicValue } from "@widgets-resources/piw-utils";
+
+declare function require(name: string): string;
```

Comment:
I see this is common throughout other widget-code. 

If we add "node" to the `types` in utils-react-widgets/configs/tsconfig.json or in a widget's tsconfig (if it overrides the base from utils) then we can avoid this declaration - and remove usages.

Thoughts? @mmehmetAliIzci  @mijay

Reply:
Idd solves the problem. Added since i think it wont introduce any breaking change